### PR TITLE
update content security policy to allow google tagging

### PR DESCRIPTION
**Pull request by hgulv1**
### JIRA link

https://tools.hmcts.net/jira/browse/CIV-446

### Change description

enable google tagging on cmc frontend

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

File: charts/cmc-citizen-frontend/Chart.yaml [hunk 1]
```diff
@@ -7,8 +7,9 @@ description: Helm chart for the HMCTS CMC Citizen Frontend service
 # chart-cmc - demo: https://github.com/hmcts/chart-cmc/tree/master/cmc
 # flux - AAT AKS: https://github.com/hmcts/cnp-flux-config/blob/365e8ee13e20d4f66c925f5a0648db26dcad8872/k8s/aat/money-claims/citizen-frontend/citizen-frontend.yaml#L15
 # prob other envs for flux ^^^
-maintainers:
-  - name: HMCTS CMC Dev Team
+maintainers:ontentSecurityPolicy.ts
```
Comment (sabahirfan) [inline charts/cmc-citizen-frontend/Chart.yaml:10]: please revert the changes in this file

**Review by asthamalviya — APPROVED** [review]

**Review by sabahirfan — APPROVED** [review]
